src/habitat.cr

```diff
@@ -37,6 +37,64 @@ class Habitat
     end
   end
 
+  # Creates settings in the embedding class or module.
```

Comment:
I like this, but what do you think about:

> Embeds settings in a Class or Module

I think it reads just a touch better

Reply:
Agree!